testsuite/smoke-test/src/rosetta.rs

```diff
@@ -2682,6 +2688,172 @@ async fn test_delegation_pool_operations() {
     }
 }
 
+#[tokio::test]
+async fn test_fee_statement() {
+    const NUM_TXNS_PER_PAGE: u16 = 2;
+
+    let (mut swarm, _, _, rosetta_client) = setup_test(
```

Comment:
Can you make these instead of `_` to `_name` similar to line 135?

If something is `_` it is dropped immediately.  If something is named `_blah` it's dropped at the end of the block (in this case the end of the function).